core/src/main/scala/org/apache/spark/internal/Logging.scala

```diff
@@ -135,7 +136,12 @@ private[spark] trait Logging {
         val replLevel = Option(replLogger.getLevel()).getOrElse(Level.WARN)
         if (replLevel != rootLogger.getEffectiveLevel()) {
           System.err.printf("Setting default log level to \"%s\".\n", replLevel)
-          System.err.println("To adjust logging level use sc.setLogLevel(newLevel).")
+          if (SparkSubmit.isRShell) {
```

Comment:
How about I change the log like `To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel)`? Thanks!